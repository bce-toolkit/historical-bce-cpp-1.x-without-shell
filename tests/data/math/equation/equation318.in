5 3
1 0 -1 -1 0
2 1 -3 -1 0
0 2 -1 0 0
# NO2+H2O=HNO3+NO
