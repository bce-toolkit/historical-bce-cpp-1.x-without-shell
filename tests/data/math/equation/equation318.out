3Xa
Xa
2Xa
Xa
